src/common/cuda_utils.h

```diff
@@ -68,6 +68,110 @@ inline __device__ bool __is_supported_cuda_architecture() {
 }
 #endif  // __CUDACC__
 
+/*!
```

Comment:
I would prefer this trunk to be reviewed and merged separately in PR: https://github.com/apache/incubator-mxnet/pull/12031